pkg/validation/test.go

```diff
@@ -486,40 +489,33 @@ func (v *Validator) validateTestConfigurationType(
 	var needsReleaseRpms bool
 	if testConfig := test.OpenshiftAnsibleClusterTestConfiguration; testConfig != nil {
 		typeCount++
-		clusterCount++
```

Comment:
These were previously used to compute "count of installed clusters", but the new validation targets the problematic state directly.